ts/Dashboard/EditMode/Sidebar.ts

```diff
@@ -72,7 +73,52 @@ class Sidebar {
     public static components: Array<Sidebar.AddComponentDetails> = [
         {
             text: 'layout',
-            onDrop: function (): void {}
+            onDrop: function (
+                sidebar: Sidebar,
```

Comment:
The `sidebar` is not used here.

Reply:
I'd leave it as it is so that we know in the future it is here. Apart from the fact, that we are using a second argument and there is no way of leaving the first argument blank.